main/vstring.c

```diff
@@ -81,12 +81,12 @@ extern vString *vStringNew (void)
 
 extern void vStringPut (vString *const string, const int c)
```

Comment:
I usually use `static inline` and put definition in the header. There are more details on the differences [here](https://gcc.gnu.org/onlinedocs/gcc/Inline.html). Without putting in the header and no `inline` keyword like above, I think it pretty much guarantees it won't be inlined. Also `extern` as used here is superfluous as it's the default for functions like this, it's akin to putting `auto int x;` for a local variable, AFAIK.